dhis-2/dhis-api/src/main/java/org/hisp/dhis/appmanager/App.java

```diff
@@ -80,6 +81,8 @@
 
     private String baseUrl;
 
+    private List<String> authorities;
```

Comment:
Maybe better to use a Set<> here? we don't really care about the ordering of these, but we want to avoid duplicates